api/endpoints/ogcapi_features.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import os
+from re import A
+import requests
+from api import settings
+from flask import request
+from flask_restplus import Resource
+from api.restplus import api
+from werkzeug.wrappers import Response
+from typing import Optional
+import json
+
+try:
+    import urllib.parse as urlparse
+except ImportError:
+    import urlparse as urlparse
+
+log = logging.getLogger(__name__)
+
+ns = api.namespace(
+    "ogcapi-features", description="Operations related to OGC API - Features"
```

Comment:
I wasn't sure if we wanted to just put the path as "features", which is pretty generic, so I made it `ogcapi-features` to be a little more specific. Through, I think we may also want to make it even more specific, like `point-cloud-features` in case there's another OGC API Features service we want to integrate in the future with different data.

Reply:
pygeoapi is capable of serving other ogc specs (actually we customized it to do point clouds). I'd be fine calling it `features`. Really all that matter is what does the final api url that is public facing look like, the less complicated the better. at the least `ogc-features` seems enough, api is redundant.

Reply:
I'll change it to just be features